docs/en/helm/known-issues.md

```diff
@@ -0,0 +1,55 @@
+# Limitations and known issues
```

Comment:
Please add a section about application upgrade.

1. What will we do when we change the values, how does the underlying helm chart be updated?
2. Using trait will patch the `main` workload, which may cause the pods of workload to restart.

Reply:
got it.